src/common/execution/order-helper.js

```diff
@@ -95,29 +93,7 @@ export const checkRequestRequirements = async (
       );
     }
   }
-  // check storage token
-  if (
-    paramsObj[IEXEC_REQUEST_PARAMS.IEXEC_RESULT_STORAGE_PROVIDER] ===
-      STORAGE_PROVIDERS.IPFS ||
-    paramsObj[IEXEC_REQUEST_PARAMS.IEXEC_RESULT_STORAGE_PROVIDER] ===
-      STORAGE_PROVIDERS.DROPBOX
-  ) {
-    const isStorageTokenSet = await checkWeb2SecretExists(
-      contracts,
-      smsURL,
-      requester,
-      getStorageTokenKeyName(
-        paramsObj[IEXEC_REQUEST_PARAMS.IEXEC_RESULT_STORAGE_PROVIDER],
-      ),
-    );
-    if (!isStorageTokenSet) {
-      throw Error(
-        `Requester storage token is not set for selected provider "${
-          paramsObj[IEXEC_REQUEST_PARAMS.IEXEC_RESULT_STORAGE_PROVIDER]
-        }". Result archive upload will fail.`,
-      );
-    }
-  }
+
```

Comment:
we want to disable the check only for ipfs, the dropbox storage token is still linked to the requester and must be checked if dropbox is selected.